x11-plugins/compiz-plugins-community/compiz-plugins-community-0.8.14.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+if [[ "${PV}" == "9999" ]]; then
```

Comment:
Why are you changing ebuild style in the middle of the series? It looks weird-ish.